pages/communityevents/_id.vue

```diff
@@ -63,31 +65,10 @@ export default {
       return this.$store.getters['communityevents/sortedList']()
     }
   },
-  created() {
-    this.groupid = this.$route.params.id
-  },
   mounted() {
     this.$store.dispatch('communityevents/clear')
-
-    if (this.groupid) {
-      // Ensure our select is set to the right value
-      this.$store.commit('group/remember', {
-        id: 'communityevents',
-        val: this.groupid
-      })
-    }
   },
-
   methods: {
-    groupChange: function(newGroup) {
```

Comment:
This page component gets much simpler too, with all this logic outsourced to the mixin...